certification/code/certificates.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+'''
+Generate a certificate from a template.
+
+* Requires the python package 'cairosvg' to be installed.
+  Please visit http://cairosvg.org/ for install instructions.
+* Some systems may also need to have 'cairo' installed.
+  Please visit http://cairographics.org/download/ for the same.
+'''
+
+import sys
+import os
+import re
+import tempfile
+import cairosvg
+from django.conf import settings
+
+#---------------------------------------------------------------
```

Comment:
PEP8 and PEP257 are some general notes on a style, but they're very helpful when you're going to write more Python and contribute to different projects, because many of them may impose PEP8-syntax style.

In AMY we don't really follow all PEP257 rules (docstrings are often in triple apostrophes `'''` instead of triple quote marks `"""`; multiline doctsrings don't always have ending `"""` on a separate line), but we do try to follow the language rule (https://github.com/swcarpentry/amy/pull/724#discussion_r59117147).